src/v2/controllers/vreplicaset_controller/proof/liveness/api_actions.rs

```diff
@@ -81,6 +81,75 @@ pub proof fn lemma_api_request_outside_create_or_delete_loop_maintains_matching_
         _ => {}
     };
 }
+
+// TODO: Prove this.
+#[verifier(external_body)]
+pub proof fn lemma_api_request_not_made_by_vrs_maintains_matching_pods(
```

Comment:
This is an important lemma. I am surprised that it's not easy to prove given that the non-interference invariant is already very strong.

Reply:
I have not tried to prove this yet --- I can try quickly this evening, falling back to comments if needed.

Reply:
Went through basically right away.